qa_filterbank is flaky

The `qa_filterbank` test has failed twice, both times on Fedora 34:

* https://github.com/gnuradio/gnuradio/runs/6312676712?check_suite_focus=true
* https://github.com/gnuradio/gnuradio/runs/6327946929?check_suite_focus=true

```
FAIL: test_000 (__main__.test_filterbank_vcvcf)
Generates nfilts sets of random complex data.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/gnuradio/gnuradio/gr-filter/python/filter/qa_filterbank.py", line 120, in test_000
    self.assertComplexTuplesAlmostEqual(comb_results2, outdata2, 6)
  File "/__w/gnuradio/gnuradio/gnuradio-runtime/python/gnuradio/gr_unittest.py", line 73, in assertComplexTuplesAlmostEqual
    return all([
  File "/__w/gnuradio/gnuradio/gnuradio-runtime/python/gnuradio/gr_unittest.py", line 74, in <listcomp>
    self.assertComplexAlmostEqual(x, y, places, msg)
  File "/__w/gnuradio/gnuradio/gnuradio-runtime/python/gnuradio/gr_unittest.py", line 37, in assertComplexAlmostEqual
    raise self.failureException(
AssertionError: (0.10920628619252344-0.17120145326118436j) != (-0.07943732291460037-0.12398221343755722j) within 6 places
```

So far I haven't been able to reproduce the problem locally.

A bug with similar symptoms was fixed recently in #5753. (The problem was due to an unaligned buffer being passed into VOLK.)

--- a/gr-filter/python/filter/qa_filterbank.py
+++ b/gr-filter/python/filter/qa_filterbank.py
@@ -111,7 +111,7 @@ class test_filterbank_vcvcf(gr_unittest.TestCase):
         fb.set_taps(taps2)
         outdata2 = None
         # Wait until we have new data.
-        while (not outdata2) or outdata[0] == outdata2[0]:
+        while (not outdata2) or abs(outdata2[0] - outdata[0]) < 1e-6:
             time.sleep(waittime)
             outdata2 = snk.level()
         self.tb.stop()

